dev-python/numba/numba-0.49.1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# enable 3.8 when llvmlite gets 3.8.
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="NumPy aware dynamic Python compiler using LLVM"
+HOMEPAGE="http://numba.pydata.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="examples openmp threads"
+
+distutils_enable_tests pytest
+
+RDEPEND="
+	>=dev-python/llvmlite-0.32.0[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	threads? ( dev-cpp/tbb )
+	openmp? ( virtual/mpi[cxx] )
+"
+BDEPEND="
+	${RDEPEND}
```

Comment:
This looks weird, but possible. But please double-check, I'll check it more thoroughly when testing this package.

Reply:
I haven't done a lot of testing for this package as their testing depends are not listed anywhere.

Reply:
You'd want to do `BDEPEND="test? ( ${RDEPEND} )" which is what `distutils_enable_tests` does already.